sdk/cosmos/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos/AppConfiguration.java

```diff
@@ -40,6 +40,9 @@ public class AppConfiguration extends AbstractCosmosConfiguration {
     @Value("${azure.cosmos.queryMetricsEnabled}")
     private boolean queryMetricsEnabled;
 
+    @Value("${azure.cosmos.maxDegreeOfParallelism:0}")
```

Comment:
Same here, do we need to expose the value itself with the property name?

Reply:
Done.